app/models/manager_refresh/inventory_collection_default/infra_manager.rb

```diff
@@ -124,7 +124,7 @@ def ems_clusters(extra_attributes = {})
     def storages(extra_attributes = {})
       attributes = {
         :model_class => ::Storage,
-        # TODO: change :manager_ref => [:location],
+        :manager_ref => [:location],
```

Comment:
So, is the location unique in a whole provider? If yes, then it's ok.

Reply:
It is unique

Reply:
Well it can be just an NFS path, https://github.com/ManageIQ/manageiq-providers-ovirt/blob/master/app/models/manageiq/providers/redhat/infra_manager/refresh/parse/parser.rb#L46

This will not be unique across providers on purpose, the idea is that it will link the same storage across providers

Reply:
Correct it is unique within single provider. Do you have a suggestion how to make sure it is unique across the providers?